specs/solid-indexing/index.html

```diff
@@ -32,43 +31,13 @@
   <section class="informative">
     <h2>Introduction</h2>
 
-    <p>The Web is made of documents like web pages, API data, images and so on that are linked together. They are
-      information that anyone with the proper authorization can access. Documents can be found everywhere
-      on the web and we often have to follow the links between them in order to find relevant information. The
-      web is therefore browsable but browsing it manually can be very long to find what we are searching for.
-      That's why search engines have been invented. They browse the web for us by following the
-      links, they find in documents so they can tell us where to get documents about a particular subject.
-    </p>
+    <p>The Semantic Web vision consists in publishing on the Web machine-readable documents so that machine, like humans, become able to browse the Web, following links from one document to another, in order to gather information and answer complex questions.</p>
 
-    <p>Search engines are doing indexing. Indexing is a largely used mechanism that allow to find data faster thanks to
-      [=indexes=]. By querying
-      the appropriate indexes one can avoid to search at unuseful places. Indexing is also widely used in all kind of software, especially in databases. On the traditionnal web, search engines are indexing  
-      documents globally so even if we saved time we still have to look at the content of the documents by ourselves to see 
-      if the thing we are searching for is there.
-    </p>
+    <p>Hovewer, when a lot of data is available, gathering information and answering complex questions becomes very ineffiscient without indexes. Indexes are made of meta-data that allow to find data easier and faster. Indexing is a widely used mechanism in all kind of software, especially in databases.</p>
 
-    <p>There is an other Web, the semantic Web, sometimes called the Web of data or the Web 3.0. It works the same as the 
-      traditionnal Web except that information is described to machines so they can "understand" what documents and the 
-      things they contain are about. This way we can directly get the information we need without having to look at the 
-      content of documents. This is made possible thanks to the Resource Description Framework (RDF). On this improved Web, 
-      knowledge can be deduced automatically by machines. For instance, if a document is about a person and the machines 
-      know that a person is a human, they can deduce that this person is a human. Search engines can benefit a lot from the web 
-      of data and some engines already take advantage of it to better respond to our queries.
-    </p>
+    <p>This document proposes an RDF vocabulary to describe these indexes, which are also intended to be published on the Web, to facilitate the search of data by Semantic Web agents. In addition, this document also presents some indexing and index-querying techniques.</p>
```

Comment:
Other kinds of indexes may co-exist.
```suggestion
    <p>This document proposes an RDF vocabulary to describe one kind of such indexes, which are also intended to be published on the Web, to facilitate the search of data by Semantic Web agents. In addition, this document also presents some indexing and index-querying techniques.</p>
```